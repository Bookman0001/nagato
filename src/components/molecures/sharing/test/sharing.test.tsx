/**
 * @jest-environment jsdom
 */
import { shallow } from 'enzyme'

import Sharing from '../'

test('Sharing', () => {
  const component = shallow(<Sharing />)
  expect(component).toMatchSnapshot()
})
