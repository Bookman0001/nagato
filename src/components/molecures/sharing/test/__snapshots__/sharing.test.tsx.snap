// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sharing 1`] = `
<styled.section>
  <styled.div>
    <ForwardRef(ShareButton-facebook)
      url="http://localhost/"
    >
      <styled.span>
        <CirclePicture
          alt="facebook"
          height={40}
          src="/assets/facebook.png"
          width={40}
        />
      </styled.span>
    </ForwardRef(ShareButton-facebook)>
    <ForwardRef(ShareButton-twitter)
      url="http://localhost/"
    >
      <styled.span>
        <CirclePicture
          alt="twitter"
          height={40}
          src="/assets/twitter.png"
          width={40}
        />
      </styled.span>
    </ForwardRef(ShareButton-twitter)>
    <styled.div>
      <Link
        href="mailto:odonel51486666@gmail.com"
      >
        <styled.span>
          <CirclePicture
            alt="mail"
            height={40}
            src="/assets/mail.png"
            width={40}
          />
        </styled.span>
      </Link>
    </styled.div>
  </styled.div>
</styled.section>
`;
